app/src/main/java/org/oppia/app/topic/TopicFragmentPresenter.kt

```diff
@@ -59,13 +61,15 @@ class TopicFragmentPresenter @Inject constructor(
   }
 
   private fun setUpViewPager(viewPager: ViewPager, topicId: String?) {
-    val adapter = ViewPagerAdapter(fragment.fragmentManager!!, topicId!!)
+    val adapter = ViewPagerAdapter(fragment.fragmentManager!!, topicId!!, storyId)
     viewPager.adapter = adapter
     tabLayout.setupWithViewPager(viewPager)
     tabLayout.getTabAt(0)!!.setText(fragment.getString(R.string.overview)).setIcon(tabIcons[0])
     tabLayout.getTabAt(1)!!.setText(fragment.getString(R.string.play)).setIcon(tabIcons[1])
     tabLayout.getTabAt(2)!!.setText(fragment.getString(R.string.train)).setIcon(tabIcons[2])
     tabLayout.getTabAt(3)!!.setText(fragment.getString(R.string.review)).setIcon(tabIcons[3])
+    if (!topicId.isEmpty() && !storyId.isEmpty())
```

Comment:
Update this to: `if (topicId.isNotEmpty() && storyId.isNotEmpty())`

Reply:
Done